### PR TITLE
feat(ci): saving docker images digests and publishing dockerhub description

### PR DESCRIPTION
**Pull request by rogeriopeixotocx**
**Proposed Changes**

We should have the docker image digests published versioned in our repository.

- on docker release saving all images digests into `docs/digests.csv` and converting it to `docs/digests.md` file
- fixing docker pulls badge link
- dependabot's commits should follow [semantic commit messages](https://gist.github.com/joshbuchea/6f47e86d2510bce28f8e7f42ae84c716)
- unifying `.github/workflows/release-dkr-image-for-tag-latest.yml` and `.github/workflows/release-dkr-image-for-tag.yml` in a single script
- updating dockerhub page after publishing new image with `docs/dockerhub.md` file versioned in this repo 
- fixing grammar `pull_request_template.md`

I submit this contribution under Apache-2.0 license.

### CODE REVIEW

File: docs/dockerhub.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+## Git Repo
+
+https://github.com/Checkmarx/kics
+
+## Command
+
+To scan a directory/file on your host you have to mount it as a volume to the container and specify the path on the container filesystem with the -p KICS parameter (see the full list of CLI options below)
+
+```txt
```
Comment (felipe-avelar) [inline docs/dockerhub.md:15]: I think ` ```sh` makes more sense here

**Review by felipe-avelar — COMMENTED** [review]
In general, its looks good to me, I think only a minor change on `dockerhub.md` should be done...

File: docs/dockerhub.md (hunk 1 above)
Comment (rogeriopeixotocx) [reply docs/dockerhub.md]: done. thanks

**Review by joaoReigota1 — APPROVED** [review]
LCTM

**Review by felipe-avelar — APPROVED** [review]
LGTM

**Pull request merged**